packages/fluentui/react/src/themes/teams/components/Icon/svg/icons/error.tsx

```diff
@@ -10,6 +10,7 @@ export default {
       />
     </svg>
   ),
+  // TODO: styles overrides
```

Comment:
This was never working in the previous implementation :\ It seems like this is the only one used in production, so I believe it is save to move this to the svgIconStyles, and later if necessary we can allow users to define additional styles to their icons.